step.yml

```diff
@@ -157,6 +157,13 @@ inputs:
       value_options:
         - "yes"
         - "no"
+  - code_signing_allowed: "no"
+    opts:
+      title: Code Signing Allowed
+      description: Whether codesigning is allowed for this build.
```

Comment:
can you extend the description to state why it should be set to NO for the simulator

Reply:
I don't actually know the answer to that 😬 What is the reason why we have it codesigning disabled by default?

Reply:
```
title: Enable code signing
description: When building an app for the simulator, code signing is not required and is set to "no" by default.
Only on rare occasions do you need to set the flag to "yes", usually when working with certain test cases or third-party dependency
```

Reply:
Thanks for the help on the description. Updated in https://github.com/bitrise-steplib/steps-xcode-build-for-simulator/pull/31/commits/bcb9fcb2c31fb1e566fa36fc9b97619bc477099d